huxley/www/js/components/AdvisorRosterView.js

```diff
@@ -204,10 +211,23 @@ var AdvisorRosterView = React.createClass({
             </Button>
           </td>
         );
+
+        const waiverCheck =
+          delegate && delegate.waiver_submitted ? '\u2611' : '\u2610';
+
+        const positionPaperCheck =
+          delegate.assignment &&
+          assignments[delegate.assignment - 1].paper &&
```

Comment:
The use of assignments[delegate.assignment - 1] only works as intended if
id of assignment object from server == key of that assignment object in the "assignments" variable

For example if you got an assignment object from the server with id = 4, your assignments variable would look like
assignments[0] = assignment(id=4)
and your call
assignments[delegate.assignemnt - 1] == assignments[3]
which would be undefined

Try adding dummy assignments in your test env to increment to PK in the DB a little bit and then test with those assignments